CHANGELOG.next.asciidoc

```diff
@@ -79,6 +79,8 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 
 *Packetbeat*
 
+- Use base-16 for reporting `serial_number` value in TLS fields {pull}41542[41542]
```

Comment:
I suggest adding some mention of ECS as the rational like "... to align with the ECS recommendation."